acceptance/lib/helper.rb

```diff
@@ -167,12 +167,7 @@ def upgrade_package(host, name)
   end
 
   def get_defaults_var(host, varname)
-    if master.is_pe?
```

Comment:
Is there an expectation that this will get fixed at some point?  Wondering if we should create a follow-on ticket for our backlog to move back to that later on if so?